modules/ossm-cr-example.adoc

```diff
@@ -18,63 +18,257 @@ This example `ServiceMeshControlPlane` definition contains all of the supported
+        - name: status-port
+          port: 15020
+        - name: http2
+          port: 80
+          targetPort: 8080
+        - name: https
+          port: 443
+          targetPort: 8443
+      meshExpansionPorts: []
+    egress: # _the_ istio-egressgateway
+      service:
+        type: ClusterIP
+        ports:
+        - name: status-port
+          port: 15020
+        - name: http2
+          port: 80
+          targetPort: 8080
+        - name: https
+          port: 443
+          targetPort: 8443
+    additionalIngress:
+      some-other-ingress-gateway: {}
+    additionalEgress:
+      some-other-egress-gateway: {}
 
+  policy:
     mixer:
-      policy:
-        autoscaleEnabled: false
+      enableChecks: false # default
+      failOpen: false # default
+      runtime:
+        container:
+          image: mixer # default mixer image
+          resources: {} # mixer resources
 
-      telemetry:
-        autoscaleEnabled: false
-        resources:
-          requests:
-            cpu: 100m
-            memory: 1G
-          limits:
-            cpu: 500m
-            memory: 4G
+  telemetry:
+    type: <istiod_#or_Mixer_or_Remote>
+    mixer: # legacy v1
+      sessionAffinity: false
+      batching:
+        maxEntries: 100
+        maxTime: 1s
+      adapters:
+        useAdapterCRDs: false
+        kubernetesenv: true
+        stdio:
+          outputAsJSON: false
+        prometheus:
+          metricsExpiryDuration: 10m
+        stackdriver: null
 
+  runtime:
     pilot:
-      autoscaleEnabled: false
-      traceSampling: 100
+      deployment:
+        replicas: 2
+      pod:
+        affinity: {}
+      container:
+        resources:
+          limits: {}
+          requirements: {}
 
-    kiali:
-      enabled: true
+  addons:
+    visualization:
+      grafana:
+        address: some-grafana-url 
+        install: 
+          selfManaged: true 
+          config:
+            env: {} 
+            envSecrets: {} 
+          persistence:
+            storageClassName: ""
+            accessMode: ReadWriteOnce
+            capacity: 5Gi 
+          service: 
+            ingress:
+              contextPath: /grafana
+              tls:
+                termination: reencrypt
+          runtime:
+            deployment: {}
+            pod: {}
+      kiali:
+        name: kiali 
+        install: # install kiali CR if not present
+          config:
+            dashboard:
+              viewOnly: false
+              enableGrafana: true
+              enableTracing: true
+              enablePrometheus: true
+          service: 
+            ingress:
+              contextPath: /kiali
+          runtime:
+            deployment: {}
+            pod: {}
+----
 
-    grafana:
-      enabled: true
+The following table lists the parameters for the `ServiceMeshControlPlane` resource.
 
-    tracing:
-      enabled: true
-      jaeger:
-        template: all-in-one
-----
+.`ServiceMeshControlPlane` resource parameters
```

Comment:
While we've presented this information in tables before, I'm not sure this _particular_ organization works very well.  I think I'd have a really hard time turning these tables into a properly formatted  and nested YAML file.  I think it might be better to do something like we did in V1, where we had a table for each chunk of the configuration file (gateways, policy, telemetry, runtime, addons).  (if we don't have time before the release, then we should definitely do it ASAP after the release.)

Reply:
I'm not sure how effective these tables are as configuration documentation. I think that they're recreations of api docs that are better hosted elsewhere. 

I think we should be writing configuration docs in the user guide rather than these tables. I think we should talk to @heathjoy about where we add that content.

Reply:
@neal-timpe let's look at this right after the release. If we have a lot of tables that make up config topics, then I agree that they should be in a guide and xref'd as appropriate.